Detections/MultipleDataSources/SucessfullPosswordSprayAttampOnAWSConsolelogin.yaml

```diff
@@ -0,0 +1,121 @@
+id: 188db479-d50a-4a9c-a041-644bae347d1f

```

Comment:
Could we also do the reverse here? A GuardDuty alert for password spray identifying an IP Address logging into Azure?

Reply:
we do not have Guard Duty alerts in RDA, without RDA testing its risky to make detection